core/lib/data/utils/compress_file_utils.dart

```diff
@@ -0,0 +1,40 @@
+
+import 'dart:typed_data';
+
+import 'package:core/utils/app_logger.dart';
+import 'package:core/utils/build_utils.dart';
+import 'package:flutter_image_compress/flutter_image_compress.dart';
+
+class CompressFileUtils {
+
+  /// Maximum image size displayed (Unit: KB)
+  static const int MAXIMUM_IMAGE_SIZE = 300;
```

Comment:
`MAXIMUM_IMAGE_SIZE_KB`

Reply:
why you choose `300 Kb` is maximum?

Reply:
IMO 300Kb is what I test to set a stable image quality. We can change it. And of course if you have any suggestions please comment here.